copilot-cli/skills/integration/SKILL.md

```
@@ -0,0 +1,193 @@
+name: integration
+description: "Merge worktree branches in dependency order, run combined tests, and verify before review. Activates when all delegate tasks complete or user runs /integrate."
+---
+
+# Integration Skill
+
+## Overview
+
+Integration phase: Merge worktree branches in dependency order, run combined tests, and verify before review.
+
+**Position in workflow:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)